include/deal.II/grid/tria_accessor.h

```diff
@@ -628,6 +628,12 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
    */
 
   /**
+   * Pointer to the @p ith vertex bounding this object. Throw an exception if
+   * <code>dim=1</code>.
+   */
+  TriaAccessor<0,dim,spacedim> vertex_accessor (const unsigned int i) const;
```

Comment:
I'm still for consistency. Why can't you just re-appropriate one of the existing constructors by using `level=0` and `index=vertex_index`? You might have to add a new constructor to the current class, but that's not a big commitment.

How does this currently work for the vertex iterators in 1d? Can't we just use the same approach?